src/richie-front/js/utils/filters/isResourceBasedFilterGroupName.ts

```diff
@@ -0,0 +1,15 @@
+import includes from 'lodash-es/includes';
```

Comment:
This is a nitpick but we add a line break between dependencies imports and local imports (... which we probably need to document in the handbook!).